smoketest.sh

```diff
@@ -76,6 +78,17 @@ function runGrafana() {
         --rm -d "${stream}:${tag}"
 }
 
+function runReportGenerator() {
+    podman run \
+        --name reports \
+        --pod cryostat \
+        --cpus 1 \
+        --memory 512M \
+        --restart on-failure \
+        --env QUARKUS_HTTP_PORT=10001 \
+        --rm -d quay.io/cryostatio/cryostat-reports:1.0.0-SNAPSHOT
```

Comment:
When running smoketest.sh with `CRYOSTAT_AUTH_MANAGER=io.cryostat.net.BasicAuthManager CRYOSTAT_DISABLE_SSL=true CRYOSTAT_DISABLE_JMX_AUTH=true sh smoketest.sh`, I'm denied access to `quay.io/cryostatio/cryostat-reports`:
```
HTTP_PORT=10001 --rm -d quay.io/cryostatio/cryostat-reports:1.0.0-SNAPSHOT
Trying to pull quay.io/cryostatio/cryostat-reports:1.0.0-SNAPSHOT...
Error: initializing source docker://quay.io/cryostatio/cryostat-reports:1.0.0-SNAPSHOT: reading manifest 1.0.0-SNAPSHOT in quay.io/cryostatio/cryostat-reports: unauthorized: access to the requested resource is not authorized
```

Reply:
Whoops, that should be `quay.io/cryostat/cryostat-reports` - I fixed the namespace.

That image repository is currently still empty on quay.io itself - I haven't pushed a manually-built image there and there is no CI auto-push set up yet, either. For now, clone `https://github.com/cryostatio/cryostat-reports` and build it locally yourself with `./mvnw package`. It should be loaded into your local podman image registry after that.

Reply:
Thanks, I can see the reports in the recording dropdowns and in a new tab now